lib/astutils.h

```diff
@@ -363,8 +363,6 @@ class FwdAnalysis {
         const Token *token;
     };
 
-    std::vector<KnownAndToken> valueFlow(const Token *expr, const Token *startToken, const Token *endToken);
```

Comment:
The last usage was removed in 921887a28122124e0f0b28130d682bfb7b5c487d - looks like a leftover.